packages/react-components/react-avatar/src/components/AvatarGroup/renderAvatarGroup.tsx

```diff
@@ -1,13 +1,28 @@
 import * as React from 'react';
 import { getSlots } from '@fluentui/react-utilities';
 import type { AvatarGroupState, AvatarGroupSlots } from './AvatarGroup.types';
+import { Popover, PopoverTrigger } from '@fluentui/react-popover';
+import { Tooltip } from '@fluentui/react-tooltip';
 
 /**
  * Render the final JSX of AvatarGroup
  */
 export const renderAvatarGroup_unstable = (state: AvatarGroupState) => {
   const { slots, slotProps } = getSlots<AvatarGroupSlots>(state);
 
-  // TODO Add additional slots in the appropriate place
-  return <slots.root {...slotProps.root} />;
+  return (
+    <slots.root {...slotProps.root}>
+      {state.root.children}
+      {state.hasOverflow && slots.popoverSurface && slots.popoverTrigger && (
```

Comment:
popoverSurface and popoverTrigger...those are user provided right? What happens if the user forgets to add these? Are they required? Is there an error? If there's no tooltip, are users going to understand why?

Reply:
Tooltip is always provided when the overflow indicator is shown. The Surface and trigger are required, but that is set by the state. If the user doesn't provide them, it renders a default one which is based on the design spec. I think the only error that maybe should be given is if the user doesn't provide Avatars or the user provides something that isn't an avatar. For now everything is rendered but I was planning on bringing that up in the jam session.